en/deploy/README.md

```diff
@@ -130,159 +130,106 @@ Your code is now on GitHub. Go and check it out!  You'll find it's in fine compa
-Successfully installed django-1.11.3
+$ pa_autoconfigure_django.py https://github.com/<your-github-username>/my-first-blog.git
 ```
 
+As you watch that running, you'll be able to see what it's doing:
 
-> **Note** The `pip install` step can take a couple of minutes.  Patience, patience!  But if it takes more than five minutes, something is wrong.  Ask your coach.
-
-<!--TODO: think about using requirements.txt instead of pip install.-->
+- Downloading your code from GitHub
+- Creating a virtualenv on PythonAnywhere, just like the one on your own PC
+- Updating your settings file with some deployment settings
+- Setting up a database on PythonAnywhere using the `manage.py migrate` command
+- Setting up your static files (we'll learn about these later)
+- And configuring PythonAnywhere to serve your web app via its API
 
-### Creating the database on PythonAnywhere
+On PythonAnywhere all those steps are automated, but they're the same steps you
+would have to go through with any other server provider.  The main thing to notice
+right now is that your database on PythonAnywhere is actually totally separate from
+your database on your own PC—that means it can have different posts and admin accounts.
 
-Here's another thing that's different between your own computer and the server: it uses a different database. So the user accounts and posts can be different on the server and on your computer.
-
-Just as we did on your own computer, we repeat the step to initialize the database on the server, with `migrate` and `createsuperuser`:
+As a result, just as we did on your own computer, we repeat the step to
+initialize the admin account with `createsuperuser`. PythonAnywhere has automatically
+activated your virtualenv for you, so all you need to do is run:
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-(mvenv) $ python manage.py migrate
-Operations to perform:
-[...]
-  Applying sessions.0001_initial... OK
-(mvenv) $ python manage.py createsuperuser
+(ola.pythonanywhere.com) $ python manage.py createsuperuser
 ```
 
-## Publishing our blog as a web app
-
-Now our code is on PythonAnywhere, our virtualenv is ready, and the database is initialized. We're ready to publish it as a web app!
-
-Click back to the PythonAnywhere dashboard by clicking on its logo, and then click on the **Web** tab. Finally, hit **Add a new web app**.
-
-After confirming your domain name, choose **manual configuration** (N.B. – *not* the "Django" option) in the dialog. Next choose **Python 3.6**, and click Next to finish the wizard.
+Type in the details for your superuser.  Best to use the same ones as you're using
+on your own computer to avoid any confusion, unless you want to make the password
+on PythonAnywhere more secure.
 
-> **Note** Make sure you choose the "Manual configuration" option, not the "Django" one. We're too cool for the default PythonAnywhere Django setup. ;-)
+Now, if you like you can also take a look at your code on PythonAnywhere using
```

Comment:
Not sure (as I'm not a native speaker of English): Wouldn't a comma belong after the "if you like"?

Reply:
yep, that'd be a good comma i think.